plugin/backend.go

```diff
@@ -162,6 +162,7 @@ func (pm *Manager) Remove(name string, config *types.PluginRmConfig) error {
 	}
 
 	pm.pluginStore.Remove(p)
+	os.RemoveAll(filepath.Join(pm.libRoot, p.PluginObj.ID))
```

Comment:
Lets try and avoid direct access to the PluginObj from manager and use a wrapper instead. Eventually, we should convert all Plugin struct fields to internal types.